src/aatoolbox/datasources/iri/iri_seasonal_forecast.py

```diff
@@ -144,12 +149,21 @@ def _process(filepath: str, ds, clobber: bool):
         return filepath
 
     def load_raw(self):
-        ds = xr.load_dataset(
-            self._get_raw_path(),
-            decode_times=False,
-            drop_variables="C",
-        )
-        return ds
+        try:
+            ds = xr.load_dataset(
+                self._get_raw_path(),
+                decode_times=False,
+                drop_variables="C",
+            )
+            return ds
+        except ValueError as err:
+            raise ValueError(
```

Comment:
@turnerm For the `iri_auth` there are two options. Either the env var is not set due to which it is None, which we catch above. However, another option is that the `iri_auth` is not valid. In that case a file is downloaded, but this actually doesn't contain any data and is not a netcdf file. I thought this was the best way to catch it, but curious to hear if you got any other ideas!